ivy/stateful/activations.py

```diff
@@ -91,7 +84,7 @@ def _forward(self, x, complex_mode=None):
         ret
             The outputs following the RELU activation *[batch_shape, d]*
         """
-        return ivy.relu(x, complex_mode=ivy.default(complex_mode, self._complex_mode))
+        return ivy.relu(x, self._complex_mode)
```

Comment:
just a minor change, but `complex_mode` is a keyword-only argument in `ivy.relu`